app/components/meeting-schedule/component.js

```diff
@@ -80,11 +81,5 @@ export default Ember.Component.extend({
             });
         }
     }),
-    selectedItemId: Ember.computed(function () {
-        // return this.get('model.items').then((results) => {
-        //     console.log(results.get('firstObject.id'));
-        //     return results.get('firstObject.id');
-        // });
-        return 2;
-    })
+    selectedItemId: 0
```

Comment:
Worth noting that this creates a 404 request to the API whenever the page is loaded but then rectifies itself almost instantaneously and makes the proper GET call.